src/components/calcite-input-message/calcite-input-message.tsx

```diff
@@ -36,9 +36,6 @@ export class CalciteInputMessage {
   /** specify the status of the input field, determines message and icons */
   @Prop({ reflect: true, mutable: true }) status: Status = "idle";
 
-  /** specify the appearance of any slotted message - default (displayed under input), or floating (positioned absolutely under input) */
-  @Prop({ reflect: true }) type: InputMessageType = "default";
```

Comment:
@driskull is this wording okay or what do you suggest?
```tsx
  /** specify the appearance of any slotted message - default (displayed under input), or floating (positioned absolutely under input)
   * @deprecated "floating" type is no longer supported
   */
  @Prop({ reflect: true }) type: "default";
```

Reply:
Yes, that sounds good.